openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -298,13 +298,15 @@ def cleaned_extended_profile(self):
 
 def get_registration_extension_form(*args, **kwargs):
     """
-    Convenience function for getting the custom form set in settings.REGISTRATION_EXTENSION_FORM.
+    Convenience function for getting the custom form set in site configurations REGISTRATION_EXTENSION_FORM.
```

Comment:
The purpose of adding site configuration support to a Django setting is to support both the site configuration and the default Django configuration. We should not remove backwards compatibility in case there are no sites configured or it is a single site installation. @Nekenhei

Reply:
Alright, I'll implement a solution that prioritize the site configurations over the platform settings.